src/AqueductV1Pair.sol

```diff
@@ -242,19 +243,19 @@ contract AqueductV1Pair is IAqueductV1Pair, AqueductV1ERC20, SuperAppBase {
      * @param _kLast The previous product of the reserves (_reserve0 * _reserve1).
```

Comment:
Does `(_reserve0 * _reserve1)` here need renaming too?

Reply:
@joshkokatnur this comment here. Also in `_calculateReservesBothFlows` and `_calculateReservesFlow1`